browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1855,9 +1855,27 @@ until they verify, or until 90 days have passed.
     <message name="IDS_ACCOUNT_RECOVERY_FAILED" desc="Brave Wallet text">
       Account recovery failed. Please try again.
     </message>
-    <message name="IDS_MAINNET" desc="Brave Wallet network selection button text">
+    <message name="IDS_MAINNET" desc="Brave Wallet network selection button text(network name)">
       Mainnet
     </message>
+    <message name="IDS_RINKEBY" desc="Brave Wallet network selection button text(network name)">
```

Comment:
Do we need to translate these strings? If not, we should add `translateable="false"` tag.